torch/_dynamo/variables/misc.py

```diff
@@ -559,7 +564,30 @@ def __init__(self, value, **kwargs):
     def call_function(
         self, tx, args: "List[VariableTracker]", kwargs: "Dict[str, VariableTracker]"
     ) -> "VariableTracker":
-        unimplemented("numpy")
+        if not config.numpy_ndarray_as_tensor or not HAS_NUMPY_TORCH_INTEROP:
+            unimplemented(f"numpy.{self.value}()")
+        import torch_np
+
+        from .builder import wrap_fx_proxy_cls
+        from .tensor import NumpyTensorVariable
+
+        options = VariableTracker.propagate([[self]], [args], [list(kwargs.values())])
```

Comment:
this is not the idiomatic way to do it (although I forget what the idiomatic way is)